subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/corefeature/TaskCreationPerformanceTest.groovy

```diff
@@ -22,7 +22,6 @@ class TaskCreationPerformanceTest extends AbstractCrossVersionPerformanceTest {
 
     def "create many tasks"() {
         given:
-        runner.testProject = "createLotsOfTasks"
```

Comment:
Does running this on any other test project make sense?

Reply:
No. Actually, we should merge this test with `JavaConfigurationPerformanceTest` and add `createLotsOfTasks ` as a test project there, WDYT?